### ISSUE TITLE
Set up result publishing

### ISSUE THREAD

**Issue by rabernat** [pangeo-data/pangeo-integration-tests#5]
As proposed by @nbren12 in https://github.com/pangeo-data/pangeo-integration-tests/issues/1#issuecomment-839063503, we will be running these tests as a kubernetes cron job. The clusters will need some way to publish their results, likely just a static http endpoint.

How should this work? I see two basic possibilities:
- Each testing cluster publishes its results to a public bucket in its own cloud storage. With this, write credentials can be packaged into the kubernetes environment.
- All clusters publish to a central location. This would require some sort of common credentials.

The former certainly seems easier.

We also need to decide what we want to log. It will be important to capture detailed logs from all parts of the stack, as well as performance reports (timing, etc).

**Comment by nbren12** [comment]
I agree (1) is easier and allows a more federated model. It would be nice to setup some kind of table of badges, we could bake this into our test script, but maybe there is an easier way.
